data/settings.html

```diff
@@ -59,39 +65,46 @@ <h2>Owie Settings</h2>
           form.setAttribute("action", oldAction);
         };
 
+        const monCheckbox = document.getElementById("monChk");
```

Comment:
This bit won't be necessary if instead of using the dataset on the checkbox you directly dump the checked attribute on it.
i.e. "<input type="checkbox" %SPOOFING_DISABLED%>" and SPOOFING_DISABLED is either "" or "checked"